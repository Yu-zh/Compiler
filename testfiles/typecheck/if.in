int main(){
    int a = 5;
    if(a){
        a = a+1;
    }
    if(a>5){
        a = a+1;
    }
    bool t = 5;
    int f = 1;
}