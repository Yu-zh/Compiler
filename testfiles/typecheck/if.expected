Error at line 3: type does not check!
Error at line 9: type does not check!
